### DISCUSSION TITLE
What does MemB stands for?

### DISCUSSION THREAD

**Discussion by chengguangnan** [aristocratos/btop#816] [Development]
Please let me know what does the B stands for. 

Thanks.

**Comment by rghvdberg** [comment]
Was wondering the same thing.
MemB : memory usage in Bytes

You can toggle between bytes and percentage via de options menu [o] then [5] , scroll down till you see `Proc mem bytes`
<img width="694" height="277" alt="afbeelding" src="https://github.com/user-attachments/assets/3ce2f700-4c54-422c-91ca-40fa0f02d65a" />
